docs/source/common/lightning_cli.rst

```diff
@@ -717,13 +762,51 @@ example can be :code:`ReduceLROnPlateau` which requires to specify a monitor. Th
 
     cli = MyLightningCLI(MyModel)
 
+
 For both possibilities of using :meth:`pytorch_lightning.utilities.cli.LightningArgumentParser.add_optimizer_args` with
 a single class or a tuple of classes, the value given to :code:`optimizer_init` will always be a dictionary including
 :code:`class_path` and :code:`init_args` entries. The function
 :func:`~pytorch_lightning.utilities.cli.instantiate_class` takes care of importing the class defined in
 :code:`class_path` and instantiating it using some positional arguments, in this case :code:`self.parameters()`, and the
 :code:`init_args`. Any number of optimizers and learning rate schedulers can be added when using :code:`link_to`.
 
+For code simplification, the LightningCLI provides properties with already registered PyTorch built-in `optimizers` and `schedulers`.
+
+.. code-block::
+
+    class MyLightningCLI(LightningCLI):
+        def add_arguments_to_parser(self, parser):
+            parser.add_optimizer_args(
+                self.registered_optimizers,
+                link_to="model.optimizer_init",
+            )
+            parser.add_lr_scheduler_args(
+                self.registered_lr_schedulers,
+                link_to="model.lr_scheduler_init",
+            )
+
+However, a user can register its own optimizers or schedulers as follow.
+
+.. code-block:: python
+
+    import torch
+    from pytorch_lightning.utilities.cli import OPTIMIZER_REGISTRIES, LR_SCHEDULER_REGISTRIES
+    from pytorch_lightning.callbacks import Callback
+
+
+    @CALLBACK_REGISTRIES
```

Comment:
```suggestion
    @OPTIMIZER_REGISTRIES
```

Reply:
Hopefully this becomes `@OPTIMIZER_REGISTRY` right?

Reply:
Yup

Reply:
Done !